src/lesson3/task10/Main.java

```diff
@@ -0,0 +1,26 @@
+package lesson3.task10;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args)
+    {
+        int f = 0;
+        int s = 1;
+        System.out.println("Сколько чисел Фибоначчи выводить?");
+          Scanner sc = new Scanner(System.in);
+          int n = sc.nextInt();
+        System.out.print("Первые " + n + " чисел последовательности Фибоначчи: ");
+        System.out.print(f + " " + s);
```

Comment:
We should place all input in the loop, because if n == 1, we will get incorrect output